src/components/layout/FooterComponent.tsx

```diff
@@ -20,7 +20,7 @@ const FooterComponent = () => {
             </p>
           </section>
           <section className='flex flex-col gap-12 text-black font-bold text-xl underline max-xl:text-base max-xl:gap-8 max-md:gap-4'>
-            <Link href='https://btctranscripts.com/' target='_blank'>
```

Comment:
Yes. For now the homepage in btctranscripts, same as `https://btctranscripts.com/`.

But instead of navigating you to another domain it stays on the new registry domain and loads the old registry homepage.

Reply:
got it!